apps/web/tsconfig.json

```
@@ -10,10 +17,13 @@
   },
   "include": [
     "next-env.d.ts",
+    "next.config.mjs",
+    "postcss.config.cjs",
     "**/*.ts",
     "**/*.tsx",
     ".next/types/**/*.ts",
-    "src/env.mjs"
+    "**/*.mjs",
```

```suggestion
    "src/env.mjs",
```
Is there a reason for this change?